lib/prediction/predictor.ts

```diff
@@ -224,7 +226,7 @@ class Worker extends events.EventEmitter {
         this._minibatchStartTime = now;
```

Comment:
can `assert(this._minibatch.length === 0)` be false when `task !=  this._minibatchTask`?

Reply:
We always flush the request before starting a new request, so the minibatch will always be empty.